ingestion_tools/scripts/data_validation/fixtures/data.py

```diff
@@ -104,63 +172,86 @@ def canonical_tomogram_metadata(canonical_tomo_meta_file: str, filesystem: FileS
 
 
 @pytest.fixture(scope="session")
-def annotation_metadata(annotation_meta_files: List[str], filesystem: FileSystemApi) -> Dict[str, Dict]:
-    """Load the annotation metadata."""
+def annotation_metadata(annotation_metadata_files: List[str], filesystem: FileSystemApi) -> Dict[str, Dict]:
+    """Load the annotation metadata. Dictionary structure: metadata = {metadata_a_filename: Dict}, metadata_b_filename: ...}."""
     metadata_objs = {}
 
-    for file in annotation_meta_files:
+    for file in annotation_metadata_files:
         with filesystem.open(file, "r") as f:
             metadata_objs[file] = json.load(f)
 
     return metadata_objs
 
 
```

Comment:
helper function below to help with ndjson data retrieval (for instance seg, point, and oriented point annotations)